sample-apps/donet-sample-app/Controllers/AppController.cs

```diff
@@ -49,7 +51,7 @@ public string OutgoingHttp()
         }
 
         [HttpGet]
-        [Route("/aws-sdk-call")]
+        [Route("/get-aws-s3-bucket")]
```

Comment:
```suggestion
        [Route("/get-aws-s3-bucket")]
```
Err, I think this was a mistake in the spec and can thus be reverted back. You were correct originally. A PR already exists to update the spec.